altair/api.py

```diff
@@ -168,15 +168,19 @@ class Layer(schema.BaseObject):
 
     @classmethod
     def from_dict(cls, dct):
+        """Create a Chart from a dict of Vega-Lite JSON."""
         return visitors.FromDict().clsvisit(cls, dct)
 
     def to_dict(self, data=True):
+        """Emit the Vega-Lite JSON for this Chart as as dict."""
         return visitors.ToDict().visit(self, data)
 
     def to_code(self, data=None):
+        """Emit the Python code required to created this Chart."""
```

Comment:
This actually returns a code object (essentially a custom AST) whose string representation is the Python code. Maybe we should make this a private method?